specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1088,6 +1130,95 @@
           }
         }
       }
+    },
+    "/keys/{key-name}/rotationpolicy": {
+      "get": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "GetKeyRotationPolicy",
+        "summary": "Lists the policy for a key.",
+        "description": "The GetKeyRotationPolicy operation returns the specified key policy resources in the specified key vault. This operation requires the keys/get permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key in a given key vault."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The key rotation policy.",
+            "schema": {
+              "$ref": "#/definitions/KeyRotationPolicy"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetKeyRotationPolicy": {
+            "$ref": "./examples/GetKeyRotationPolicy-example.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "keys"
```

Comment:
Should be uppercase since this gets used in code generation and may cause duplication.

Reply:
updated